examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Btw, how do you prevent some modules from being published, if you still want them verified in your builds?

Using `maven-deploy-plugin` `<skip>true</skip>` is suggested in [maven faq#skip](https://maven.apache.org/plugins/maven-deploy-plugin/faq.html#skip)